ci/test/00_setup_env_win64.sh

```diff
@@ -16,4 +16,4 @@ export GOAL="deploy"
 # Prior to 11.0.0, the mingw-w64 headers were missing noreturn attributes, causing warnings when
 # cross-compiling for Windows. https://sourceforge.net/p/mingw-w64/bugs/306/
 # https://github.com/mingw-w64/mingw-w64/commit/1690994f515910a31b9fb7c7bd3a52d4ba987abe
-export BITCOIN_CONFIG="--enable-reduce-exports --disable-gui-tests CXXFLAGS=-Wno-return-type"
+export BITCOIN_CONFIG="--enable-reduce-exports --enable-external-signer --disable-gui-tests CXXFLAGS=-Wno-return-type"
```

Comment:
> ci: remove --enable-external-signer from win64 job

Why this is being reverted? This is already applied globally to the CI (and I'd say pointlessly so).

Reply:
Thanks! Dropped.

Reply:
Great, can you remove it from the global config here as well. Given it's not required or actually testing anything.

Reply:
> Great, can you remove it from the global config here as well. Given it's not required or actually testing anything.

I agree. Implemented.